pkgs/development/python-modules/m3u8/default.nix

```diff
@@ -1,32 +1,39 @@
-{ lib, buildPythonPackage, fetchFromGitHub, fetchpatch, requests, iso8601, bottle, pytestCheckHook }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, iso8601
+, bottle
+, pytestCheckHook
+}:
 
 buildPythonPackage rec {
   pname = "m3u8";
-  version = "0.9.0";
+  version = "3.4.0";
 
   src = fetchFromGitHub {
     owner = "globocom";
     repo = pname;
     rev = version;
-    hash = "sha256-EfHhmV2otEgEy2OVohS+DF7dk97GFdWZ4cFCERZBmlA=";
+    hash = "sha256-jfCmvAb7bF6nYFNUPXVG61x0RiO4vcyR+x7WzgPRLxI=";
   };
 
-  patches = [
-    # Fix hardcoded /tmp dir (fix build on Hydra)
-    (fetchpatch {
-      url = "https://github.com/globocom/m3u8/commit/cf7ae5fda4681efcea796cd7c51c02f152c36009.patch";
-      hash = "sha256-SEETpIJQddid8D//6DVrSGs/BqDeMOzufE0bBrm+/xY=";
-    })
+  propagatedBuildInputs = [
+    iso8601
   ];
 
-  propagatedBuildInputs = [ requests iso8601 ];
+  nativeCheckInputs = [
+    bottle
+    pytestCheckHook
+  ];
 
-  nativeCheckInputs = [ bottle pytestCheckHook ];
+  disabledTests = [
+    "test_load_should_create_object_from_uri"
```

Comment:
Short comment why those were disabled would be nice

Reply:
Added, thanks for reviewing!